platform/linuxbsd/export/export_plugin.cpp

```diff
@@ -303,6 +303,12 @@ Error EditorExportPlatformLinuxBSD::fixup_embedded_pck(const String &p_path, int
 	return OK;
 }
 
+void EditorExportPlatformLinuxBSD::get_platform_features(List<String> *r_features) const {
+	EditorExportPlatformPC::get_platform_features(r_features);
+	r_features->push_back("linux");
+	r_features->push_back("bsd");
+}
```

Comment:
I'm not sure if this is only used to get a list of `possible` tags from that platform or the `actual` ones.

Reply:
I found out by experiment that this method should return the **actual** feature tags of the platform. The export happens for the platform (`linuxbsd`), not for some sub-platform (`linux`, `bsd`, `freebsd`, `openbsd`, `netbsd`).